### PR TITLE
Tag picker

### PR DESCRIPTION
**Pull request by JUtupe**

### CODE REVIEW

**Comment by JUtupe** [comment]
widzę że tu będzie jeszcze jeden bug z `Warning: validateDOMNesting(...): <form> cannot appear as a descendant of <form>.`

poprawię to na dniach

File: src/components/ui/TagPicker/TagPicker.tsx [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+    label: string
+    onRemoveClick: () => void
+}
+
+const TagComponent: React.FC<TagComponentProps> = ({label, onRemoveClick}) => (
+    <div className={styles.tag}>
+        {label}
+        <Close className={cs(styles.close, styles.pointer)} onClick={onRemoveClick}/>
+    </div>
+)
```
Comment (DiD3n) [inline src/components/ui/TagPicker/TagPicker.tsx]: move this to separate file
react is about splitting big things into small things

File: src/components/ui/TagPicker/TagPicker.tsx [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+    label: string
+    onClick: () => void
+}
+
+const DropdownOption: React.FC<DropdownOptionProps> = ({icon, label, selected, onClick}) => (
+    <div className={cs(styles.dropdownOption, {[styles.selected]: selected})} onClick={() => onClick()}>
+        {label}
+        {icon}
+    </div>
+)
```
Comment (DiD3n) [inline src/components/ui/TagPicker/TagPicker.tsx]: same as above

**Review by DiD3n — CHANGES_REQUESTED** [review]
if possible try to separate logic as well

File: src/components/ui/TagPicker/TagPicker.tsx (hunk 2 above)
Comment (JUtupe) [reply src/components/ui/TagPicker/TagPicker.tsx]: done

**Pull request merged**